resources/rgbscripts/flames.js

```diff
@@ -0,0 +1,172 @@
+/*
+  Q Light Controller Plus
+  empty.js
+
+  Copyright (c) Branson Matheson
+  Based on work https://slicker.me/javascript/fire/fire.htm
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0.txt
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+*/
+
+// Development tool access
+var testAlgo;
+
+(
+  function()
+  {
+    var util = new Object;
+    util.initialized = false;
+
+    var algo = new Object;
```

Comment:
Codacy has a fix for the issue: [Replace `··var·algo·=·new·Object` with `var·algo·=·new·Object()`](https://app.codacy.com/gh/mcallegari/qlcplus/pullRequest?prid=9305478)
```suggestion
  var algo = new Object();
```